### ISSUE TITLE
[2: Contrast] low contrast needed by some

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-showcase-examples#67]
Although WCAG currently only addresses high contrast, some people need low luminance/luminosity (brightness) because they are extremely sensitive to light. http://w3c.github.io/low-vision-a11y-tf/requirements.html#light-sensitivity

I'm not sure how we handle this in the video. Certainly we need to address it in the accompanying web page.

**Comment by nitedog** [comment]
Not sure if we should re-interpret or go beyond what WCAG currently says.

**Comment by yatil** [comment]
I agree keeping the scope to WCAG.now – we an always amend the pages with information once the extension is published.

**Comment by shawna-slh** [comment]
We should have http://w3c.github.io/low-vision-a11y-tf/requirements.html on TR wayyy before an approved WCAG extension and I think it will be important to address it in the accompanying web page right away.

**Comment by yatil** [comment]
Good point!

**Comment by nitedog** [comment]
Will come back to this when we come to write the blurbs.

**Comment by nitedog** [comment]
Currently using the wording "adequate contrast between foreground and background". There isn't too much more space for getting into such details. Please re-open issue if you have suggestions.
